### ISSUE TITLE
What's the URL to see the list of requested cards?

### ISSUE THREAD

**Issue by melvinzhang** [firemind/ProjectFiremind-Issues#43]
I lost the link after clearing my browser history. I will record it on the Magarena wiki to prevent something like this from happening again.

**Comment by firemind** [comment]
It is part of the Status page here: https://www.firemind.ch/status

**Comment by melvinzhang** [comment]
Thanks @firemind! I seem to remember the page used to show how many requests was made for a card, is this information still shown somewhere?

**Comment by firemind** [comment]
Looks like I had to remove that part in December last year because the library I was using to track/calculate the values was incompatible with a framework update (https://github.com/twitter/activerecord-reputation-system)

I decided to remove it instead of rewriting it from scratch because it seemed the feature wasn't used. Apparently I was wrong since you at least remembered it :smile: 

So if you say this information was relevant to the magarena team I will consider rewriting it myself (the data is still there, I just can't easily display it at the moment). I'd probably also try to make the voting more obvious to users.
